src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -442,34 +510,286 @@ public int getID()
 
     /**
      * Increment the mobs killed by this colony.
-     * <p>
      * Will award achievements for mobs killed.
      */
     public void incrementMobsKilled()
     {
         killedMobs++;
         final int mobKills = this.getKilledMobs();
-        if (mobKills >= NUM_MOBS_ACHIEVEMENT_FIRST)
+        if (mobKills >= NUM_ACHIEVEMENT_FIRST)
         {
             this.triggerAchievement(ModAchievements.achievementKillOneMob);
         }
-        if (mobKills >= NUM_MOBS_ACHIEVEMENT_SECOND)
+        if (mobKills >= NUM_ACHIEVEMENT_SECOND)
         {
             this.triggerAchievement(ModAchievements.achievementKill25Mobs);
         }
-        if (mobKills >= NUM_MOBS_ACHIEVEMENT_THIRD)
+        if (mobKills >= NUM_ACHIEVEMENT_THIRD)
         {
             this.triggerAchievement(ModAchievements.achievementKill100Mobs);
         }
-        if (mobKills >= NUM_MOBS_ACHIEVEMENT_FOURTH)
+        if (mobKills >= NUM_ACHIEVEMENT_FOURTH)
         {
             this.triggerAchievement(ModAchievements.achievementKill500Mobs);
         }
-        if (mobKills >= NUM_MOBS_ACHIEVEMENT_FIFTH)
+        if (mobKills >= NUM_ACHIEVEMENT_FIFTH)
         {
             this.triggerAchievement(ModAchievements.achievementKill1000Mobs);
         }
     }
+    /**
+     * Increment the ores mined by this colony.
+     * Will award achievements for ores mined.
+     */
+    public void incrementOresMined()
+    {
+        minedOres++;
+        final int Ores = this.getMinedOres();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00117)